meta-luneos/recipes-core/busybox/busybox-static_1.36.1.bb

```diff
@@ -5,6 +5,7 @@ require recipes-core/busybox/busybox_${PV}.bb
 S = "${WORKDIR}/busybox-${PV}"
 SRC_URI += " \
             file://static-build.cfg \
+            file://enable-devmem.cfg \
```

Comment:
did you forget to add the file? I don't see it even in scarthgap based build

ERROR: /OE/build/luneos-whinlatter/openembedded-core/meta/recipes-core/busybox/busybox_1.37.0.bb: Unable to get checksum for 
              busybox SRC_URI entry enable-devmem.cfg: file could not be found